docs/explore/datadog/index-datadog.mdx

```diff
@@ -71,7 +72,7 @@ Since Redis is popular due to performance, generally you would expect most opera
 |   Action   | Method     |
 | ---------- | ----------- | 
 | Increase resources | The database can be scaled up online by going to the Web UI and enabling clustering on the database. In extreme cases, more nodes can be added to the cluster and resources rebalanced |
-| Inefficient queries | Redis allows you to view a slow log with a tunable threshold. It can be viewed either in the Redis Enterprise UI or by running:   redis-cli -h HOST -p PORT -a PASSWORD SLOWLOG GET 100 |
+| Inefficient queries | Redis allows you to view slow logs with a tunable threshold. It can be viewed either in the Redis Enterprise UI or by running:   redis-cli -h HOST -p PORT -a PASSWORD SLOWLOG GET 100 |
```

Comment:
```suggestion
| Inefficient queries | Redis allows you to view slow logs with a tunable threshold. It can be viewed either in the Redis Enterprise UI or by running:   `redis-cli -h HOST -p PORT -a PASSWORD SLOWLOG GET 100` |
```